UserManagementSystem/src/Core/Entities/User.cs

```diff
@@ -4,14 +4,6 @@ namespace UserManagementSystem.Src.Core.Entities;
 
 public class User
 {
-    public User(string firstName, string lastName, string email, string password)
-    {
-        this.FirstName = firstName;
-        this.LastName = lastName;
-        this.Email = email;
-        this.Password = password;
-    }
-    [Key]
```

Comment:
?

Reply:
no haven't we aggreed to this ? it's needed here the constructor does nothing and also [key] ef automatically recoginze it without this data annotation

Reply:
yes but this removes everything even the constructor

Reply:
yes we said the constructor here is useless?